README.rst

```diff
@@ -21,6 +21,24 @@ Then you should use ``pip`` or ``setuptools`` to install the ``zbarlight`` wrapp
 How To use ZbarLight
 ====================
 
+*The new way:*
+
+.. code-block:: python
+
+    from PIL import Image
+    import zbarlight
+
+    file_path = './tests/fixtures/one_qr_code.png'
+    with open(file_path, 'rb') as image_file:
+        image = Image.open(image_file)
+        image.load()
+
+    code = zbarlight.scan_codes('qr', image)
+    print('QR codes: %s' % code.decode())
```

Comment:
This looks like a (too) quickly adapted version of the "previous way" :) : 
    1. The code fails because `scan_codes()` return a list, not a string.
    2. Perhaps we should use `two_qr_codes.png`.